src/wr_hsi_sensing/nodes/gps_forwarding_node.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+## @file
+# @brief Node responsible for mapping Ardusimple GPS data to CoordinateMsg
+# @ingroup wr_hsi_sensing
+# @defgroup wr_hsi_sensing_gps GPS
+# @details Subscribes to the /gps/fix topic and publishes the latitude and longitude data to the /gps_coord_data topic
+# @{
+
+import rospy
+from sensor_msgs.msg import NavSatFix
+from wr_hsi_sensing.msg import CoordinateMsg
+
+
+def gps_callback(msg):
+    gps_coord_data = CoordinateMsg()
+    gps_coord_data.latitude = msg.latitude
+    gps_coord_data.longitude = msg.longitude
+
+    print("latitude : {}".format(msg.latitude))
+    print("longitude: {}".format(msg.longitude))
```

Comment:
These debugging print statements should be removed.

Reply:
Resolved in e8ff00b36f144b13af4bb8c9e64d543b2c6469c3